_security-plugin/index.md

```diff
@@ -60,7 +60,7 @@ OpenSearch includes other features that complement the security infrastructure.
 ### Dashboards multi-tenancy
 
 One such feature is OpenSearch Dashboards multi-tenancy. Tenants are work spaces that include visualizations, index patterns, and other Dashboards objects. Multi-tenancy allows for the sharing of tenants among users of Dashboards and leverages OpenSearch roles to manage access to tenants and safely make them available to others.
-For more information on creating tenants, see [OpenSearch Dashboards multi-tenancy]({{site.url}}{{site.baseurl}}/security-plugin/access-control/multi-tenancy/).
+For more information on creating tenants, see [OpenSearch Dashboards multi-tenancy]({{site.url}}{{site.baseurl}}/security-plugin/multi-tenancy/multi-tenancy-config/).
```

Comment:
For the sake of consistency, it's probably best to point to the tenant-index.md file:

({{site.url}}{{site.baseurl}}/security-plugin/security-plugin/multi-tenancy/tenant-index/)